models/question.js

```diff
@@ -39,6 +39,17 @@ module.exports = function(sequelize, DataTypes) {
                     },
                     as: "createdByUser"
                 });
+
+                /*
+                 * Ready for usage in later development
+                 *  
+                Question.belongsToMany(models.Tag, {
```

Comment:
Mislim da je bolje koristiti belongsToMany posto ce u tom slucaju Tag imati metode nad objetkima getQuestions, setQuestions, addQuestion i addQuestions, a Question isto to samo umesto Questions/Question ce imati Tags/Tag. Tako pise u dokumentaciji za hasMany nista nisu nveli.